app/src/main/java/com/example/lhi06/residue/ReviewAdapter.java

```diff
@@ -0,0 +1,87 @@
+package com.example.lhi06.residue;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by lhi06 on 24/07/2017.
+ */
+
+final class ReviewAdapter extends RecyclerView.Adapter<ReviewAdapter.ReviewViewHolder> {
+
+    private final Picasso picasso;
+    private List<AlbumReview> reviews = new ArrayList<>();
+    private ReviewClickListener clickListener;
+
+    ReviewAdapter(@NonNull final Picasso picasso) {
```

Comment:
ive tried to make this fix but the onBindViewHolder needs picasso and i cant pass through the context due to its superclass as the method isnt ran by its superclass when the parameters are changed